src/models/ReflectionLikes.js

```diff
@@ -0,0 +1,34 @@
+import { firestore} from '../firebase'
+
+export const createReflectionLikes = (id, uidLike)=>{
+  var washingtonRef = firestore.collection('reflectionResponses').doc(id);
+  washingtonRef.get().then((doc) => {
+    if (doc.exists) {
+        //console.log("Document data:", doc.data().reflectionId);
+        console.log(doc.data().numLikes)
+        console.log()
```

Comment:
reminder to self: remove all the console.log calls later.

Reply:
oops thank u